docs/source/developer/contributing.rst

```diff
@@ -80,51 +80,54 @@ case, please prepend ``[WIP]`` to your pull request's title.
 Build environment
 '''''''''''''''''
 
-Numba has a number of dependencies (mostly `Numpy <http://www.numpy.org/>`_
+Numba has a number of dependencies (mostly `numpy <http://www.numpy.org/>`_
 and `llvmlite <https://github.com/numba/llvmlite>`_) with non-trivial build
 instructions.  Unless you want to build those dependencies yourself, we
-recommend you use `Conda <http://conda.pydata.org/miniconda.html>`_ to
+recommend you use `conda <http://conda.pydata.org/miniconda.html>`_ to
 create a dedicated development environment and install precompiled versions
 of those dependencies there.
 
-First add the Binstar ``numba`` channel so as to get development builds of
-the llvmlite library::
+First add the Anaconda Cloud ``numba`` channel so as to get development builds
+of the llvmlite library::
 
    $ conda config --add channels numba
 
 Then create an environment with the right dependencies::
 
-   $ <path_to_miniconda>/conda create -n numbaenv python=3.5 llvmlite numpy
+   $ conda create -n numbaenv python=3.6 llvmlite numpy scipy
```

Comment:
To get full feature support, IIRC `scipy`, `ipython`, `jinja2` and `cffi` are needed. Also once the thread layer stuff goes in `tbb` and on OSX `llvm-openmp` and `intel-openmp`. For AOT and compiling the source a working C/C++ compiler is needed. These are available from compiler packages like `gcc_linux-64` and `gxx_linux-64` on linux and `clang_osx-64` and `clangxx_osx-64` on OSX, windows needs an appropriate MSVC version.